### ISSUE TITLE
Screen reader accessibility is missing

### ISSUE THREAD

**Issue by okevinalmeida** [r0x0r/pywebview#565]
### Specification
- pywebview version: 
- platform / version: 

### Description  
  
  Describe your issue
What the title says, I can run the example app just fine, however my screen reader can't focus on anything that appears on the app screen.
 
### Practicalities

- YES/NO I am willing to work on this issue myself.

- YES/NO I am prepared to support this issue financially.

**Comment by r0x0r** [comment]
What platform?
There was a similar discussion regarding CEF/Windows in here.

**Comment by okevinalmeida** [comment]
Indeed it is windows, more specificaly the latest build of windows 10

**Comment by r0x0r** [comment]
Duplicate https://github.com/r0x0r/pywebview/issues/545
